cmd/did-method-rest/startcmd/start.go

```diff
@@ -54,13 +54,18 @@ const (
 
 	sidetreeReadTokenFlagName  = "sidetree-read-token"
 	sidetreeReadTokenEnvKey    = "SIDETREE_READ_TOKEN"
-	sidetreeReadTokenFlagUsage = "The sidetree read token " +
+	sidetreeReadTokenFlagUsage = "The sidetree read token." +
 		" Alternatively, this can be set with the following environment variable: " + sidetreeReadTokenEnvKey
 
 	sidetreeWriteTokenFlagName  = "sidetree-write-token"
 	sidetreeWriteTokenEnvKey    = "SIDETREE_WRITE_TOKEN" //nolint: gosec
-	sidetreeWriteTokenFlagUsage = "The sidetree write token " +
+	sidetreeWriteTokenFlagUsage = "The sidetree write token." +
 		" Alternatively, this can be set with the following environment variable: " + sidetreeWriteTokenEnvKey
+
+	enableSignaturesFlagName  = "enable-signatures"
+	enableSignaturesEnvKey    = "ENABLE_SIGNATURES"
+	enableSignaturesFlagUsage = "Enable signatures. Possible values [true] [false]. Defaults to false if not set." +
```

Comment:
default value is true

Reply:
Are you saying the default value _should_ be true? Because I believe in the code I've written it defaults to false...

Reply:
yes

Reply:
Done